extensions/3DTILES_metadata/README.md

```diff
@@ -0,0 +1,256 @@
+              }
+            }
+          },
+          "classes": {
+            "_WFF_DATASET": {
+              "properties": {
+                "wff_version": {
+                  "type": "STRING"
+                },
+                "runtime_enabled": {
+                  "type": "BOOLEAN"
+                },
+                "geometry_model": {
+                  "type": "ENUM",
+                  "enumType": "GEOMETRY_MODEL_TYPE"
+                },
+                "content_type": {
+                  "type": "ENUM",
+                  "enumType": "CONTENT_TYPE"
+                },
+                "capture_platform": {
+                  "type": "ENUM",
+                  "enumType": "CAPTURE_PLATFORM_TYPE"
+                },
+                "capture_sensor": {
+                  "type": "ENUM",
+                  "enumType": "CAPTURE_SENSOR_TYPE"
+                },
+                "provider": {
+                  "type": "STRING"
+                },
+                "id": {
+                  "type": "STRING"
+                },
+                "created": {
+                  "type": "STRING"
+                },
+                "semantic": {
+                  "type": "STRING"
+                },
+                "featureClasses": {
+                  "type": "ARRAY",
+                  "componentType": "STRING",
+                  "optional": true
+                }
+              }
+            },
+            "NDVI": {
+              "properties": {
+                "NDVI": {
+                  "type": "UINT8",
+                  "normalized": true
+                }
+              }
+            },
+            "LULC": {
+              "properties": {
+                "Name": {
+                  "type": "STRING"
+                },
+                "Color": {
+                  "type": "ARRAY",
+                  "componentType": "UINT8",
+                  "componentCount": 3
+                }
+              }
+            }
+          }
+        }
+      },
+      "metadata": {
+        "tileset": {
+          "name": "Dataset",
+          "class": "_WFF_DATASET",
+          "properties": {
+            "wff_version": "1.0",
+            "runtime_enabled": true,
+            "geometry_model": "BARE_EARTH",
+            "content_type": "MESH",
+            "capture_platform": "SATELLITE",
+            "capture_sensor": "ELECTRO_OPTICAL",
+            "provider": "Maxar",
+            "id": "2e7444d7-e816-48c6-8786-c24ab354422e",
+            "created": "2020-10-05T13:46:46.672632Z",
+            "semantic": "_WFF_DATASET",
+            "featureClasses": ["NDVI", "LULC"]
+          }
+        }
+      }
+    }
+  },
+  "geometricError": 500,
+  "root": {
+    "boundingVolume": {
+      "region": [-1.2419, 0.7395, -1.2415, 0.7396, 0, 20.4]
+    },
+    "geometricError": 500,
+    "refine": "REPLACE",
+    "content": {
+      "uri": "content.gltf"
+    }
+  }
+}
+
+```
+
+
+## Property Reference
```

Comment:
What's this section for?

Reply:
Since we're no longer planning on adding the schema in Markdown, I have removed this.